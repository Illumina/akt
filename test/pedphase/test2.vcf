chr1	48513588	.	T	G	1156	PASS	.	GT	1|1	1|1	1|0
chr1	48513739	.	TGGGG	T,TGG	1772	PASS	.	GT	1|1	1|1	1|2
chr1	48516337	.	G	A	978	PASS	.	GT	1|1	1|1	1|0
