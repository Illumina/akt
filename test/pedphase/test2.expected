#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	CephChild-12882	CephMother-12878	CephFather-12877
chr1	48513588	.	T	G	1156	PASS	.	GT	1|1	1|1	1|0
chr1	48513739	.	TGGGG	T,TGG	1772	PASS	.	GT	1|1	1|1	1|2
chr1	48516337	.	G	A	978	PASS	.	GT	1|1	1|1	1|0
